.github/workflows/R-CMD-CHECK-release.yml

```diff
@@ -1,10 +1,12 @@
+# This GHA checks rOpenScPCA using dependencies from the release Bioconductor version
```

Comment:
Since it isn't necessarily only Bioconductor that is affected here. 
```suggestion
# This GHA checks rOpenScPCA using dependencies using current packages from CRAN and Bioconductor
```

Reply:
Note also that you should probably remove the R version below and let it float with the newest there too.